static/manifest.json

```diff
@@ -27,5 +27,12 @@
 	"devtools_page": "register.html",
 	"host_permissions": ["*://*/*"],
 	"permissions": ["activeTab", "scripting"],
-	"web_accessible_resources": [{ "matches": ["*://*/*"], "resources": ["courier.js"] }]
+	"web_accessible_resources": [{ "matches": ["*://*/*"], "resources": ["courier.js"] }],
+
+	"browser_specific_settings": {
+		"gecko": {
+			"id": "firefox-devtools@svelte.dev",
```

Comment:
This feels a bit long and so doesn't look that nice to me when written out. How about just `svelte.dev`?

Reply:
I might've tried that and it failed as it needs to either be a GUID or string formatted like an email address per https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_specific_settings#firefox_gecko_properties

I can try and see if `@svelte.dev` works by tomorrow

Reply:
Ah, interesting. If we have to use a `@` then I might suggest `devtools@svelte.dev`. Just `@svelte.dev` looks a bit funny to me

Reply:
I did use `devtools@svelte.dev` at first when I tried to publish it on addons.mozilla.org, but it conflicts with the old one as we discussed in Discord. Reuploading it for the "distributing on your own" doesn't work since the ID has been registered, and we can't reuse the ID according to their policy, so we need another unique ID

![image](https://github.com/sveltejs/svelte-devtools/assets/8156777/ba79211a-b90f-4767-826c-eb9e0bc78b6e)

It does seem like it's only used internally, so a random GUID like `"{daf44bf7-a45e-4450-979c-91cf07434c3d}"` would also do the job, and `@svelte.dev` seems to work too if we want to opt for a really short one.

Reply:
Oh, I thought it was just plain `svelte` that the old one was using as it's id

Reply:
Alright, I'll merge this so that we can get a signed add-on install file for Firefox on the next release